### ISSUE TITLE
strange noise

### ISSUE THREAD

**Issue by kaedewang** [VoidXH/Cavern#102]
DRP doesn't have such noise
content: F1 and Horizon
Render target: 7.1.4
F1:About 7.5s
![图片](https://user-images.githubusercontent.com/67850015/228106330-0b9542eb-d214-4085-b9dd-f7ff9930de26.png)

Horizon:About 1:44.0s
![图片](https://user-images.githubusercontent.com/67850015/228106206-1a85bcb8-dde0-4ae9-aaee-e0a78ce90d7d.png)

**Comment by kaedewang** [comment]
audiosphere:About 46.5s
![图片](https://user-images.githubusercontent.com/67850015/228108829-c4c5d282-1b17-4042-8e77-cf174420c425.png)

**Comment by VoidXH** [comment]
This is caused by #17 and was muted after #95. Please update your Cavernize, and it will give you a message if and why this happens.

**Comment by kaedewang** [comment]
I update Cavernize. There is no message and issue is still there.
audiosphere:About 46.5s
How to mute issue?
![图片](https://user-images.githubusercontent.com/67850015/228399454-f4a23ed6-2da7-436e-889d-5662a90ef8f9.png)

**Comment by kaedewang** [comment]
F1 and Horizon report the issue but audiosphere not.

**Comment by VoidXH** [comment]
Yes, indeed it's there, but the reason is unchanged. It's because of the undocumented JOC, and this is a phase alignment issue after reverting to normal JOC. It could also be mutable, but is not a priority, since the movie don't use the undocumented JOC mode.

**Comment by AVX22** [comment]
Tested a Dolby demo yesterday with Cavernize's latest update and i noticed the sparse JOC popup. When i tested, some miliseconds are muted and then there is sound again troughout the whole audio. I assume that is how its intented to work but it is very noticeable.
Will you keep developing that feature or there is no way to fix that because of the lack of documentation?

**Comment by VoidXH** [comment]
There is no way to fix that, but since no movie was coded that way, they'll work normally.

**Comment by ValZapod** [comment]
>When i tested, some miliseconds are muted and then there is sound again troughout the whole audio.

So that means only those miliseconds has sparse coding or you just do not fully remove all sparse coding? BTW, it is possible to find the delta with DRP and you will guess how it should behave?

**Comment by AVX22** [comment]
> There is no way to fix that, but since no movie was coded that way, they'll work normally.

I tested a couple of movies and the Sparse JOC pop up appeared. But after checking it, the objects are ok at the moment.

**Comment by VoidXH** [comment]
Some movies have it, but they only use it for silence, because it takes less bits.

**Comment by VoidXH** [comment]
> There is a mistake in documentation. AFAIK it was fixed in [#230](https://github.com/VoidXH/Cavern/issues/230)

It was made tolerable. Now it's seemingly a mono mix of all objects at regular volume, this is what the standard code goes down to.

**Comment by bluewidy** [comment]
[https://drive.google.com/file/d/1tRzx6-Fw2KZhKRGUTRrlnaXe_4zZq9Gd/view?usp=sharing](https://drive.google.com/file/d/1tRzx6-Fw2KZhKRGUTRrlnaXe_4zZq9Gd/view?usp=sharing)
I just got ADM BWF from a Call of Duty game trailer with DD+Atmos using Cavernize, and I can hear a very loud crackling noise so often. and what I've noticed is that while the ADM BWF from TrueHD Atmos has no LFE in the 7.1.2 bed, but the ADM BWF from DD+Atmos has LFE in the 7.1.2 bed.

**Comment by ValeZAA** [comment]
> https://drive.google.com/file/d/1tRzx6-Fw2KZhKRGUTRrlnaXe_4zZq9Gd/view?usp=sharing I just got ADM BWF from a Call of Duty game trailer with DD+Atmos using Cavernize, and I can hear a very loud crackling noise so often. and what I've noticed is that while the ADM BWF from TrueHD Atmos has no LFE in the 7.1.2 bed, but the ADM BWF from DD+Atmos has LFE in the 7.1.2 bed.

7.1.2 bed? Are you sure, typically both Truehd Atmos and DD+ Atmos uses LFE only as bed.

**Comment by bluewidy** [comment]
But this doesn't really matter to me at all. What does matter to me is that generating ADM BWF from DDP Atmos frequently introduces very loud crackling noises.
